packages/sheets/src/commands/mutations/set-worksheet-col-width.mutation.ts

```diff
@@ -56,6 +56,33 @@ export const SetWorksheetColWidthMutationFactory = (
     };
 };
 
+export const createUndoMutationsByRedos = (
```

Comment:
This name is considered "too big" for what it actually does, especially when it is exported from the package index.ts.